src/Streetcred.Sdk/Runtime/DefaultSchemaService.cs

```
@@ -43,6 +43,66 @@ public class DefaultSchemaService : ISchemaService
             return schemaRecord.SchemaId;
         }
 
+        /// <inheritdoc />
+        public async Task<string> LookupSchemaFromCredentialDefinitionAsync(Pool pool, Wallet wallet, string submitterDid,
+            string credentialDefinitionId)
+        {
+            var credDef = await LookupCredentialDefinitionAsync(pool, wallet, submitterDid, credentialDefinitionId);
+
+            if (string.IsNullOrEmpty(credDef))
+                return null;
+
+            try
+            {
+                var schemaSequenceId = Convert.ToInt32(JObject.Parse(credDef)["schemaId"].ToString());
+                return await LookupSchemaAsync(pool, wallet, submitterDid, schemaSequenceId);
+            }
+            catch (Exception) { }
```

Swallowed exceptions lose the context and information. I would utilize `Int32.TryParse` and throw an `ArgumentException`. Add a TODO to update that when we do error handling.